node/config/def.go

```diff
@@ -188,6 +188,15 @@ func DefaultStorageMiner() *StorageMiner {
 			TerminateControl:   []string{},
 			DealPublishControl: []string{},
 		},
+
+		DAGStore: DAGStoreConfig{
+			// zero value paths for transients, indices, datastore.
+			// by default they'll end up under the node repo.
```

Comment:
It wasn't immediately clear to me what this comment was referring to (until I went and looked at the DAGStoreConfig struct)

Reply:
I'll revise!